dev-requirements.txt

```diff
@@ -42,14 +42,6 @@ idna==2.10
     # via
     #   -c requirements.txt
     #   requests
-importlib-metadata==1.7.0
```

Comment:
I ran pip-compile again for no reason. I will check what’s happening

Reply:
I still haven't figured this out. I keep rollbacking those changes atm.

Reply:
I've figured it out: I am using 3.7 Python. You are probably both using 3.8 or 3.9 since import-metadata is since version 3.8 part of the stdlib.

I will resolve this convo